src/ceph-disk/ceph_disk/main.py

```diff
@@ -212,6 +212,7 @@ def is_dmcrypt(ptype, name):
     'upstart',
     'sysvinit',
     'systemd',
+    'openrc',
```

Comment:
this commit involves at least three changes:

- put the openrc support back into ceph-disk
- run py3.4 instead of py3.5 in run-tox
- address the comment regarding the `os.path.isabs()` versus `startswith('/')`.

could you reset this change and squash the related changes into the corresponding commits?